official/vision/beta/projects/yolo/dataloaders/yolo_input.py

```diff
@@ -0,0 +1,511 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Detection Data parser and processing for YOLO."""
+import tensorflow as tf
+import numpy as np
+from official.vision.beta.projects.yolo.ops import preprocessing_ops
+from official.vision.beta.projects.yolo.ops import box_ops as box_utils
+from official.vision.beta.ops import preprocess_ops
+from official.vision.beta.dataloaders import parser, utils
+
+
+def _coco91_to_80(classif, box, areas, iscrowds):
```

Comment:
Is this function necessary? can we just use 91 classes?

Reply:
makes the models smaller and it makes it easier to match the results to the paper.